WTWSMS/common/traits/02_traits.txt

```diff
@@ -1093,10 +1095,15 @@ schemer = {
 theologian = {
 	lifestyle = yes
 	
-	learning = 2
-	intrigue = 1
+	learning = 4
+	diplomacy = 2
```

Comment:
He is an expert theologian. His life is dedicated to total and absolute religious understanding. He is brilliant in the concept of religion, though I agree with the diplomacy thing.